### PR TITLE
Helm charts

### PR DESCRIPTION
**Pull request by subhommitra**
Helm charts created for ScorpioBroker deployment comprising env vars for kafka, postgres, and keycloak

### CODE REVIEW

**Review by sysarcher — CHANGES_REQUESTED** [review]
Please do not modify `gateway/` in this PR (which is related to Helm)... you can open a new PR with the changes if they're necessary

File: gateway/Cargo.toml [hunk 1]
```diff
@@ -10,13 +10,13 @@ edition = "2018"
...
-env_logger = "0.8.4"
+env_logger = "0.9.0"
 
 base64 = "0.13"
 clap = "3.0.0-beta.5"
 tokio = { version = "1.0", features = ["macros", "rt-multi-thread", "time"] }
-futures = "0.3.0"
-libc = "0.2.0"
+futures = "0.3.18"
+libc = "0.2.108"
```
Comment (sysarcher) [inline gateway/Cargo.toml:19]: `Cargo.toml` shouldn't be modified here

**Review by sysarcher — CHANGES_REQUESTED** [review]

**Comment by subhommitra** [comment]
@sysarcher no modifications have been made to `gateway` directory. The gateway directory was removed when we created a new branch from `main` and merging it would have deleted it from `main` as well. So we added it back in the latest commit. The contents of the directory remain unchanged.

**Comment by sysarcher** [comment]
@subhommitra I [highlighted the changes in the review](https://github.com/IndustryFusion/DigitalTwin/pull/40/files/eec7bb92fab55afab96a266f8790a58fd154db08#r758099620)

File: gateway/Cargo.toml (hunk 1 above)
Comment (subhommitra) [reply gateway/Cargo.toml]: Reverted these changes in a new commit

Comment (sysarcher) [reply gateway/Cargo.toml]: @subhommitra please make sure that `Cargo.lock` is also unaffected. 

Maybe, it's better to rebase to `main` branch (or merge `main` into this branch) and accept changes from `main` for files under `gateway/`

**Review by sysarcher — DISMISSED** [review]

File: gateway/Cargo.toml (hunk 1 above)
Comment (subhommitra) [reply gateway/Cargo.toml]: Merged `main` into this branch and pushed. Please review.

Comment (sysarcher) [reply gateway/Cargo.toml]: LGTM!

**Review by wagmarcel — APPROVED** [review]

**Pull request merged**